<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsHostFS</class>
 <widget class="SettingsHostFS" name="SettingsHostFS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Host Shared Folders</string>
  </property>
 <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="toolbarLayout">
     <item>
      <widget class="QPushButton" name="buttonAdd">
       <property name="text">
        <string>Add shared folder…</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRemove">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="controls" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelFolder">
        <property name="text">
         <string>Folder:</string>
        </property>
       </widget>
      </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEditFolder">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="buttonBrowse">
       <property name="text">
        <string>Browse…</string>
       </property>
      </widget>
     </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelCapacity">
        <property name="text">
         <string>Capacity (MiB):</string>
        </property>
       </widget>
      </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditCapacity">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="labelBus">
       <property name="text">
        <string>Bus:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="comboBoxBus"/>
     </item>
     <item row="1" column="4">
      <widget class="QComboBox" name="comboBoxChannel"/>
     </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelFS">
        <property name="text">
         <string>Filesystem:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxFs"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="labelOS">
        <property name="text">
         <string>OS Level:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="comboBoxOs"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelLayout">
        <property name="text">
         <string>Disk Layout:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBoxLayout"/>
      </item>
      <item row="4" column="0" colspan="4">
       <widget class="QLabel" name="labelFsHint">
        <property name="text">
         <string>Tip: For capacities ≥ 2048 MiB, FAT32 is recommended for best compatibility.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <customwidgets>
  <customwidget>
   <class>SettingsHostFS</class>
   <extends>QWidget</extends>
   <header>qt_settingshostfs.hpp</header>
  </customwidget>
 </customwidgets>
 <connections>
  <connection>
   <sender>tableView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>SettingsHostFS</receiver>
   <slot>onTableRowChanged(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
